frame_metadata/Codec.ts

```diff
@@ -6,7 +6,7 @@ export type DeriveCodec = (typeI: number) => $.Codec<unknown>;
 
 /**
  * All derived codecs for ZSTs will use this exact codec,
- * so `derivedCodec === $null` is true iff the type is a ZST.
```

Comment:
This was not a typo – `iff` in logic is an abbreviation for "if and only if", which is the most accurate description of the contract of `$null`.

Reply:
Ahh, apologies